payment-manager/src/main/resources/application.yaml

```diff
@@ -1,3 +1,6 @@
+webclient:
+  base-url: http://localhost:8083
```

Comment:
Think bigger. This "base-url" is only base for notifications. But you have transactions as well. Maybe you need to add another layer to separate urls?

Reply:
I don't quite know which way to think

Reply:
```
webclient:
  notification-manager:
       base-url: http://localhost:8083
  transaction-history:
     base-url:
  
  
```